ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/V2/InMemory.hs

```diff
@@ -85,18 +86,21 @@ newInMemoryLedgerTablesHandle ::
   , CanUpgradeLedgerTables l
   , SerializeTablesWithHint l
   ) =>
+  Tracer m V2.FlavorImplSpecificTrace ->
   SomeHasFS m ->
   LedgerTables l ValuesMK ->
   m (LedgerTablesHandle m l)
-newInMemoryLedgerTablesHandle someFS@(SomeHasFS hasFS) l = do
+newInMemoryLedgerTablesHandle tracer someFS@(SomeHasFS hasFS) l = do
   !tv <- newTVarIO (LedgerTablesHandleOpen l)
+  traceWith tracer V2.TraceLedgerTablesHandleCreate
```

Comment:
I seem to recall we already had these kind of traces.

Reply:
Ah actually we have "some" datatype for traces but we don't emit it anywhere as it is useless.
```haskell
data FlavorImplSpecificTrace
  = FlavorImplSpecificTraceInMemory
  | FlavorImplSpecificTraceOnDisk
  deriving (Show, Eq)
```

Perahaps we can even delete it...